R/ridge.R

```diff
@@ -21,7 +21,7 @@
 #'          \item{"synw"}{The synth weights(for estimating the bias)}
 #' }
 fit_ridgeaug_formatted <- function(wide_data, synth_data,
-                                   Z=NULL, lambda=NULL, ridge=T, scm=T) {
+                                   Z=NULL, lambda=NULL, ridge=T, scm=T, lambda_min_ratio = 0.001, n_lambda = 20) {
```

Comment:
It would be good to also have `lambda_max` be an optional argument where the default is NULL, then below if it's null you can set `lambda_max <- 1 + sqrt(sum((X_1 - t(as.matrix(apply(X_c, 2, mean))))^2))^2`

Reply:
Also can you add the new arguments to the doc string?

Reply:
We also might want to make `holdout_length` an optional argument

Reply:
Added this functionality. Also fixed a minor bug in the lamda_max setting (should be (1+norm)^2 instead of 1+norm^2).